bridges/YoutubeBridge.php

```diff
@@ -58,10 +58,44 @@ class YoutubeBridge extends BridgeAbstract {
 				'type' => 'number',
 				'title' => 'Maximum duration for the video in minutes',
 				'exampleValue' => 10
-			)
+			),
+			'attach_format' => array(
+				'name' => 'Attach videos to the feed as',
+				'type' => 'list',
+				'values' => array(
+					'None' => '',
+					'MP4 Video' => 'video/mp4',
+					'WebM Video' => 'video/webm',
+					'3GP Video' => 'video/3gpp',
+					'FLV Video' => 'video/x-flv',
+					'MP3 Audio' => 'audio/mp3',
+					'MP4 Audio' => 'audio/mp4',
+					'AAC Audio' => 'audio/aac',
+					'Vorbis Audio' => 'audio/ogg',
+					'Opus Audio' => 'audio/opus',
+					'FLAC Audio' => 'audio/flac',
+					'WAV Audio' => 'audio/wav'
+				),
+				'defaultValue' => '',
+				'title' => 'Tries to download the video in the format specified. Needs youtube-dl installed. Audio formats also need ffmpeg installed.'
+			),
 		)
 	);
 
+	const FILE_TYPES = array(
```

Comment:
Similar to my comment about `PARAMETERS` above: I would have liked to have only one list of all the available formats so it’s impossible to update only one. But that would have meant assembling the params definition in code, which is not possible with `const`.

Reply:
Done